openmc/deplete/results.py

```diff
@@ -394,6 +395,8 @@ def export_to_materials(
             If not provided, nuclides from the cross_sections element of
             materials.xml will be used. If that element is not present,
             nuclides from openmc.config['cross_sections'] will be used.
+        path : PathLike
+            Path to file to write. Defaults to 'materials.xml'.
```

Comment:
The method below doesn't write to materials.xml. Instead, it uses an _existing_ materials.xml as the basis for generating an updated `Materials` object. The description here needs to be changed to reflect that.